small:
  distance_angle_regulator:
    ros__parameters:
      kp_distance: 3.35
      ki_distance: 0.1
      kd_distance: 0.26
      d_term_filter_distance: 0.1

      kp_angle: 1.7
      ki_angle: 0.04
      kd_angle: 0.6
      d_term_filter_angle: 0.1

      distance_goal_tolerance: 0.005
      angle_goal_tolerance: 0.018

      control_frequency: 100.0